.eslintrc.cjs

```diff
@@ -47,7 +47,7 @@ module.exports = {
 			},
 		},
 		{
-			files: "**/*.md/*.ts",
+			files: ["**/*.md/*.ts", "**/*.md/*.js"],
```

Comment:
[Refactor] Alternately, if the snippets are switched to `\`\`\`ts\`, there'll be no need for this.

```suggestion
			files: "**/*.md/*.ts",
```